datahub-web-react/src/app/ingestV2/source/IngestionSourceTableColumns.tsx

```diff
@@ -93,25 +109,24 @@ export function NameColumn({ type, record, onNameClick }: NameColumnProps) {
                 <Icon icon="Plugs" source="phosphor" size="2xl" color="gray" />
             )}
             <DisplayNameContainer>
-                <SourceNameText
-                    ellipsis={{
-                        tooltip: {
-                            title: record.name,
-                            color: 'white',
-                            overlayInnerStyle: { color: colors.gray[1700] },
-                            showArrow: false,
-                        },
-                    }}
-                    onClick={(e) => {
-                        if (onNameClick) {
-                            e.stopPropagation();
-                            onNameClick();
-                        }
-                    }}
-                    $shouldUnderline={!!onNameClick}
+                <Tooltip
+                    title={record.name}
+                    color="white"
+                    overlayInnerStyle={{ color: colors.gray[1700] }}
+                    showArrow={false}
```

Comment:
so this would make it so that we always show a tooltip on the name even if we're not truncating right? is that desired?